### PR TITLE
Added my profile (AnuragK.json)

### PR DESCRIPTION
**Pull request by hfr58gb**
## Changes proposed

Add my profile 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
image

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1692"></a>

### CODE REVIEW

File: public/data/AnuragK.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      "url": "https://twitter.com/AnuragS41695054",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Hashnode",
+      "url": "https://anuragk24.hashnode.dev",
+      "icon": "hashnode"
+    },
+    {
+      "name": "Follow me on Stackoverflow",
```
Comment (Panquesito7) [inline public/data/AnuragK.json:23]: I recommend capitalizing the `o` here. What do you think?
```suggestion
      "name": "Follow me on StackOverflow",
```

P.S.: You may need to make a PR to add the StackOverflow icon. 🙂


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Comment (hfr58gb) [reply public/data/AnuragK.json]: yeah, updated it

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🚀 
You (or someone else) need to make a PR to add the StackOverflow, then this PR can be successfully merged. Let us know if you need any help!


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Review by kumarsonsoff3 — APPROVED** [review]
Thanks for contributing! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Comment by hfr58gb** [comment]
thanks Priyanshu [😊](https://emojipedia.org/smiling-face-with-smiling-eyes/)

**Pull request merged**